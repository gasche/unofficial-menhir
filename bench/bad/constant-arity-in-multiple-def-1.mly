%start<int> a
%%

a: {}

%public b(X): {}
