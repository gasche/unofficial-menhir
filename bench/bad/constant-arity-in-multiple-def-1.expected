File "constant-arity-in-multiple-def-2.mly", line 2, characters 8-9:
File "constant-arity-in-multiple-def-1.mly", line 6, characters 8-9:
Error: symbol b is defined with arities 2 and 1.
